security/db_router.py

```diff
@@ -31,3 +31,33 @@ def allow_relation(self, obj1, obj2, **hints):
 
     def allow_migrate(self, db, app_label, model_name=None, **hints):
         return True
+
+
+class MultipleDBSecurityLoggerRouter:
+    """
+    A router controlling tha all that read and write database operations will be performed via mirror without atomic.
```

Comment:
A router controlling **that all** read and write database operations **for apps specified in `route_app_labels`** will be performed via mirror without atomic.